src/strategies/blockzerolabs-cryptonauts/index.ts

```diff
@@ -0,0 +1,90 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { ethers } from 'ethers';
```

Comment:
Hi @umarsa everything looks good but would be great if you can change this import to below
```js
import { getAddress } from '@ethersproject/address';
```